app/models/concerns/gobierto_budgets/budget_line_algolia_helpers.rb

```diff
@@ -45,10 +45,10 @@ def algolia_as_json
           kind: kind,
           resource_path: resource_path,
           class_name: self.class.name
-        }.merge(translated_attributes)
+        }.merge(algolia_translated_attributes)
       end
 
-      def translated_attributes
+      def algolia_translated_attributes
```

Comment:
Why not remove algolia from the file name and from the method?

Reply:
This entire helper file is going to be removed after cleaning the algolia dependencies, but this method name conflicts with another one added by the new code

Reply:
Al the algolia references has been deleted in this branch